handlers/desired_lrp_handlers_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	"code.cloudfoundry.org/bbs/models"
 	"code.cloudfoundry.org/bbs/models/test/model_helpers"
 	. "code.cloudfoundry.org/bbs/test_helpers"
+	mfakes "code.cloudfoundry.org/diego-logging-client/testhelpers"
```

Comment:
**Suggestion**

I think `metronFakes` is a more descriptive name. What do you think?

Reply:
Following the naming convention from the other tests in the project.